storage_test.go

```diff
@@ -35,6 +35,10 @@ import (
 	main "github.com/RedHatInsights/ccx-notification-writer"
 )
 
+// WrongDatabaseDriver is any integer value different from DBDriverSQLite3 and
+// DBDriverPostgres
+const WrongDatabaseDriver = 10
```

Comment:
does it have to be exported?

Reply:
it's in *test* namespace only. We'll very probably split the `storage_test.go` later. But will make it local.

Reply:
fixed